core/src/test/scala/kafka/utils/TestInfoUtils.scala

```diff
@@ -39,6 +39,12 @@ object TestInfoUtils {
       } else {
         throw new RuntimeException(s"Unknown quorum value")
       }
+    } else if (testInfo.getDisplayName().contains("groupProtocol=")) {
```

Comment:
Could explain why this is needed?

Reply:
It's because the parameterized tests in `PlaintextConsumerTest` choose between "generic" and "consumer". The new consumer ("consumer") is only supported for KRaft clusters, so it's necessary to make the test use KRaft in this instance.

Reply:
I've replaced this in the new commit.